P/proxTV/build_tarballs.jl

```diff
@@ -2,26 +2,27 @@ using BinaryBuilder, Pkg
 using Base.BinaryPlatforms
 
 name = "proxTV"
-version = v"3.3.0"
+version = v"3.3.1"
 
 # Collection of sources required to build proxTV.
 sources = [
-    GitSource("https://github.com/albarji/proxTV", "0b8973cd7e1a2348c842393f689949eeebc9f654"),
+    GitSource("https://github.com/amontoison/proxTV", "6f7ad7e1048a6d97aa1e92ec35937ae4f155b1a0"),
```

Comment:
@amontoison This is no longer the official repository. Did you submit your changes to the original proxTV?

Reply:
I forgot to submit the changes.
I opened this PR: https://github.com/albarji/proxTV/pull/62

Otherwise, we can't use proxTV in Julia.